ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+               (ansible_local.pki.enabled|d() | bool) }}'
+
+                                                                   # ]]]
+# .. envvar:: prosody__pki_realm_path [[[
+#
+# Directory path where PKI realm live.
+prosdoy__pki_realm_path: '{{ ansible_local.pki.path
+                            if (ansible_local|d() and ansible_local.pki|d() and
+                                ansible_local.pki.path|d())
+                            else "/etc/pki/realms" }}'
+                                                                   # ]]]
+# .. envvar:: prosody__pki_realm [[[
+#
+# Default PKI realm to use.
+prosody__pki_realm: '{{ ansible_local.pki.realm
+                       if (ansible_local|d() and ansible_local.pki|d() and
+                           ansible_local.pki.realm|d())
+                       else "domain" }}'
+                                                                   # ]]]
+# .. envvar:: prosody__pki_crt_filename [[[
+#
+# Default CRT file name to use.
+prosody__pki_crt_filename: '{{ ansible_local.pki.crt
+                              if (ansible_local|d() and ansible_local.pki|d() and
+                                  ansible_local.pki.crt|d())
+                              else "default.crt" }}'
+                                                                   # ]]]
+# .. envvar:: prosody__pki_key_filename [[[
+#
+# Default private key file name to use.
+prosody__pki_key_filename: '{{ ansible_local.pki.key
+                              if (ansible_local|d() and ansible_local.pki|d() and
+                                  ansible_local.pki.key|d())
+                              else "default.key" }}'
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_name [[[
+#
+# Name of the hook script which will be stored in hook directory.
+prosody_pki_hook_name: 'prosody'
+
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_path [[[
+#
+# Directory with PKI hooks.
+prosody_pki_hook_path: '{{ ansible_local.pki.hooks
+                         if (ansible_local|d() and ansible_local.pki|d() and
+                             ansible_local.pki.hooks|d())
+                         else "/etc/pki/hooks" }}'
+
+                                                                   # ]]]
+# .. envvar:: prosody_pki_hook_action [[[
+#
+# Specify how changes in PKI should affect prosody, either 'reload' or 'restart'.
+prosody_pki_hook_action: 'reload'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# .. envvar:: prosody__admins [[[
+#
+# list of admins jabber accounts of this prosody instance
+prosody__admins: []
+                                                                   # ]]]
+# .. envvar:: prosody__config_global: [[[
+#
+# main prosody configuration
+# @see prosody.im
```

Comment:
Since this will be included in the role documentation on https://docs.debops.org/, you should include at least a full URL for the documentation here, https://prosody.im/doc/configure